neuro_evolution_ctrnn/tools/trainer_cma_es.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import os
+from pathlib import Path
+from typing import Callable
+from deap import base
+from deap import creator
+from deap import tools
+from deap import cma
+from tools import algorithms
+from tools.configurations import TrainerCmaEsCfg
+from tools.helper import write_checkpoint, get_checkpoint
+
+
+class TrainerCmaEs(object):
```

Comment:
Wie schon oben erwähnt, finde ich die Bezeichnung Trainer nicht gut, sondern besser den Begriff Optimierer verwenden.

Reply:
> Guter Punkt. Bei den CTRNNs muss man bedenken, dass diese

hab's geändert. Das CTRNN wird jetzt innerhalb der schleife erstellt

> Wie schon oben erwähnt, finde ich die Bezeichnung Trainer nicht gut, sondern besser den Begriff Optimierer verwenden

Stimmt. Das werde ich jetzt gleich ändern.

Reply:
das wort "trainer" kommt jetzt in dem gesamten repository nicht mehr vor